### PR TITLE
Factor out xqueue vars into own vars file, plus minor changes

### PR DESCRIPTION
**Pull request by jbau**
@jrbl @sefk

### CODE REVIEW

File: playbooks/edx-west/prod-app.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@
     # this indicates the path to site-specific (with precedence)
     # things like nginx template files
     local_dir:  '../../../configuration-secure/ansible/local'
+    not_prod: false
```
Comment (jbau) [inline playbooks/edx-west/prod-app.yml:21]: needed with ansible 1.3

File: playbooks/edx-west/stage-ansible.cfg [hunk 2]
```diff
@@ -8,7 +8,7 @@ hash_behaviour=merge
 # These are environment-specific defaults
 forks=10
 #forks=1
-log_path=stage-edx-ansible.log
+log_path=~/stage-edx-ansible.log
```
Comment (jbau) [inline playbooks/edx-west/stage-ansible.cfg:11]: i keep getting permissions issues, so just moving it to my homedir

File: playbooks/edx-west/prod-xqueue.yml [hunk 3]
```diff
@@ -10,7 +10,7 @@
     # things like nginx template files
     local_dir:  '../../../configuration-secure/ansible/local'
   vars_files:
-    - "{{ secure_dir }}/vars/edxapp_prod_vars.yml"
+    - "{{ secure_dir }}/vars/xqueue_prod_vars.yml"
```
Comment (jbau) [inline playbooks/edx-west/prod-xqueue.yml:13]: see recent commits in edx-secrets repo

**Comment by jrbl** [comment]
I haven't tested, but you have?

It reads good to me and and I like all of these changes. :+1: any time testing confirms it actually works.

**Comment by jbau** [comment]
tested.

**Pull request merged**